scripts/globals/msg.lua

```diff
@@ -216,6 +216,7 @@ msgBasic =
     ADD_EFFECT_HP_HEAL     = 167, -- Additional effect: The <player> recovers <number> HP.
     ADD_EFFECT_DISPEL      = 168, -- Additional effect: <target>'s <Status Effect> effect disappears!
     ADD_EFFECT_WARP        = 169, -- Additional effect: Warp! (used by Halloween staves)
+    LEVEL_RESTRICTED       = 314; -- <target>'s level is currently restricted to 0. Equipment affected by the level restriction will be adjusted accordingly.
```

Comment:
Shouldn't this comment indicate that it is restricted to <number> so that it is apparent that passing a value with alter the message.

Reply:
Sure can. I had copy pasted it straight from message.log but it makes sense to modify it like you said.